ci/ray_ci/upgrade_version.py

```diff
@@ -0,0 +1,96 @@
+import click
+import os
+import subprocess
+from typing import Set
+
+bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
+
+non_java_files = {
+    "ci/ray_ci/utils.py",
+    "python/ray/_version.py",
+    "src/ray/common/constants.h",
+}
+
+DEFAULT_NON_JAVA_VERSION = "3.0.0.dev0"
+DEFAULT_JAVA_VERSION = "2.0.0-SNAPSHOT"
```

Comment:
change these to `MASTER_BRANCH_VERSION` and `MASTER_BRANCH_JAVA_VERSION`:

- use master branch, it is not really the "default"
- and drop the "non java", here and everywhere. that is the main version. the string used in java is a special thing.

Reply:
done

Reply:
should I also replace `non_java_files` with something else or just leave it like that?